packages/cli/src/config/schema.ts

```diff
@@ -881,7 +882,7 @@ export const schema = {
 			location: {
 				doc: 'Log file location; only used if log output is set to file.',
 				format: String,
-				default: path.join(UserSettings.getUserN8nFolderPath(), 'logs/n8n.log'),
+				default: path.join(Container.get(InstanceSettings).n8nFolder, 'logs/n8n.log'),
```

Comment:
Or move to `InstanceSettings`? Not 100% sure where the line should be though, but I like the thought of having this centralized.

Reply:
we can revisit this later. Maybe there could be another file in `cli` that manages all n8n subfolders.